### ISSUE TITLE
Tweets not loading even in default Tweet! webpage

### ISSUE THREAD

**Issue by stoyicker** [StanScates/Tweet.js-Mod#7]
Hello, I'm trying to deploy a webpage using this mod for the Tweet! plugin http://tweet.seaofclouds.com/

After many unsuccessful attempts to deploy it directly on my webpage and with my custom folder structure, I decided to give a try to use the Tweet! plugin webpage (so I didn't have to specify the modpath option). However, same result is obtained. I'm using:

Windows 7 Professional x86
Apache HTTP Server 2.2.22.
PHP 5.2.17 VC6 Thread-Safe (already checked it's working with phpinfo test).

I've downloaded the /twitter/ folder from the mod and placed it in the root of htdocs, as well as replaced in the index.php the data for OAuth to work.
Then, also in the root of htdocs, I have the /tweet/ folder which contains the Tweet! plugin webpage. I've changed the JS reference from jquery.tweet.cs to http://localhost:57500/twitter/jquery.tweet.js (Apache is listening to port number 57500). Debug in index.php is set to true.

When I try to load the webpage in either Firefox or Chrome (lastest version of both), in the console I get 404 for http://localhost:57500/twitter/ (which obviously exists, and can be browsed to from both the filesystem and the Internet browser), and inifinite 'loading tweets' message. Same behavior for Internet Explorer 9, but no message in the console.

Anyone can confirm if this is a bug/problem or is it hapenning only to me?

**Comment by corporatereports** [comment]
I think we may have had the same issue but I found a solution and hopefully it will work for you too. Initially I had my  url as http://localhost:8888/twitter/ and it didn't work at all. After an hour or so I decided to change the directory to this: http://localhost:8888/tweetTest/twitter/ and in my index.html file I made the modpath: '/tweetTest/twitter/' and it worked.

**Comment by StanScates** [comment]
I'm sure this is related to the non-standard port configuration.
I will investigate as soon as I am able

**Comment by stoyicker** [comment]
I'll try your workaround @corporatereports but, in the meantime, @StanScates I discovered this:

When I was deploying your mod the first time (when you were helping me and finally we found out I didn't have set up Apache to work with PHP) I used version 1.3.1, and I finally got it to work with Apache listening to port 57500 and the modPath set. However now I'm trying to deploy 1.4. I'm going to try 1.3.1 with current configuration and collect results.

**EDIT:** Same behavior with v1.3.1. @corporatereports fix reports an error in jQuery (I'm using the same jQuery file used by the original Tweet!.js webpage, 1.6.4). Which version are you using?
**EDIT2:** I've tried the development version of jQuery 1.6.4 and the error is thrown in this block:

```
// Do send the request
// This may raise an exception which is actually
// handled in jQuery.ajax (so no try/catch here)
xhr.send( ( s.hasContent && s.data ) || null );
```

Also tried jQuery 2.0.0. Thrown in same block.
